.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
what is this file for ?

Reply:
C'est pour garantir les mêmes indentations et formatage de code pour tous les développeurs.